storage/imxrt-flash/flashdrv.h

```diff
@@ -19,8 +19,8 @@
 
 #include "rom_api.h"
 
-#define FLEXSPI_DATA_ADDRESS 0x60000000
-#define FLEXSPI2_DATA_ADDRESS 0x70000000
+#define FLASH_EXT_DATA_ADDRESS 0x60000000
+#define FLASH_INTERNAL_DATA_ADDRESS 0x70000000
```

Comment:
Maybe we should stick with FLEXSPI1, FLEXSPI2 nomenclature? We should be able to generalize this driver for both imxrt1060 and imxrt1170 families.